rpcs3/rpcs3qt/pad_led_settings_dialog.cpp

```diff
@@ -0,0 +1,98 @@
+﻿#include "pad_led_settings_dialog.h"
+#include <QColorDialog>
+
+pad_led_settings_dialog::pad_led_settings_dialog(QDialog* parent)
+    : QDialog(parent), ui(new Ui::pad_led_settings_dialog)
+{
+	ui->setupUi(this);
+	setFixedSize(size());
+
+	connect(ui->b_save, SIGNAL(released()), this, SLOT(close_dialog()));
+
+	connect(this, SIGNAL(pass_led_settings(int, int, int, bool, bool, int)), parent, SLOT(apply_led_settings(int, int, int, bool, bool, int)));
```

Comment:
See above

Reply:
Thank you, I used pointers instead (313190b):

```
connect(this, &pad_led_settings_dialog::pass_led_settings, qobject_cast<pad_settings_dialog*>(parent), &pad_settings_dialog::apply_led_settings);
```

However, this now throws an exception if `parent` is null or if it isn't `pad_settings_dialog`. It's not really important as it should never happen, but if this dialog is ever reused elsewhere, it will be. Should this be handled in any way?